sql/moz-fx-data-shared-prod/telemetry_derived/mozregression_aggregates_v1/query.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
```

Comment:
I'd argue that the most appropriate home for this table is in the `org_mozilla_mozregression_derived` dataset rather than in telemetry_derived.

We could put the view under `org_mozilla_mozregression`, though arguably it will be more discoverable under `telemetry`.

Reply:
Makes sense, I'll change this.